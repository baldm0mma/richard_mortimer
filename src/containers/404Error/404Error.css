.error-page {
  color: #7edf7e;
  font-family: 'VT323', monospace;
  font-size: 3rem;
  margin-top: 200px;
  text-align: center;
  text-shadow: #7edf7e 1px 0 10px;
}