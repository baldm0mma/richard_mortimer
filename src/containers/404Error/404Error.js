import './404Error.css';
import React from 'react';
import Typist from 'react-typist';
import { Link } from 'react-router-dom';

export const Error = () => {
  return (
    <Typist>
      <div>
        <h2 className='error-page'> 404: Page not found </h2>
        <Link to='/criminals'>
          <p className='back_to_database'>Click here to return to database</p>
        </Link>
      </div>
    </Typist>
  );
};
