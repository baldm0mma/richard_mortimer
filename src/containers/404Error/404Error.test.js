import { Error } from './404Error';
import { shallow } from 'enzyme';
import React from 'react';

describe('404 Error', () => {
  it('should render the 404 Error Component if the user attempts to navigate to a page that does not exsist', () => {
    const wrapper = shallow(<Error />);
    expect(wrapper).toMatchSnapshot();
  });
});