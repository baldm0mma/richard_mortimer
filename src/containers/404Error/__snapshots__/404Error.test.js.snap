// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`404 Error should render the 404 Error Component if the user attempts to navigate to a page that does not exsist 1`] = `ShallowWrapper {}`;
